.github/ISSUE_TEMPLATE/release.md

```diff
@@ -148,6 +148,10 @@ TODO: Add guidelines for labeling issues.
 - [ ] If this is a new minor version in the newest major version then run `./docs/governance/templates/release.sh {source version tag} latest`.
 - [ ] Copy the files from `build/release/` generated from `make release` to the release draft you created.  You can drag and drop the files using the Github UI.
 - [ ] Open the [`README.md`](readme-deploy) update the version references and submit. (Release candidates can ignore this step.)
+- [ ] Pun proto-gen-doc to update API references in open-match-docs repo.
+- [ ] Update Slack invitation link.
```

Comment:
Please give a little more info on how.

Reply:
Adding minikube run here too in release instructions? Ofcourse the how there would again be important :) May be as we do these things for this release, we update this doc :p and check it in the last thing.